plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp

```diff
@@ -97,6 +97,8 @@ class read_only {
       //string                  recent_slots;
       //double                  participation_rate = 0;
       optional<string>        server_version_string;
+      string                  core_symbol;
+      uint64_t                core_symbol_precision;
    };
```

Comment:
These need to be optional to support working with diff versions of nodeos and cleos.

Reply:
These add additional fields to JSON output. AFAIK this struct doesn't cross the wire in binary form, so adding fields shouldn't be a problem.

Reply:
Got it. Also, they've been updated to optional. Thanks